pootle/apps/pootle_project/models.py

```diff
@@ -224,8 +228,11 @@ def accessible_by_user(cls, user):
 
         :param user: The ``User`` instance to get accessible projects for.
         """
-        username = 'nobody' if user.is_anonymous() else user.username
-        key = iri_to_uri('projects:accessible:%s' % username)
+        if user.is_superuser:
+            key = iri_to_uri('projects:all')
```

Comment:
Can't this (and probably more code below) be simplified to `return cls.objects.cached_dict(user).values()`? The project codes are already in the cache, no need to add a separate entry :)